content/release-notes/1.19.0.md

```diff
@@ -0,0 +1,39 @@
+  * Added support for troubleshoot.sh/v1beta2 SupportBundle specs.
+{{</features>}}
+
+{{<changes>}}
+  * The KOTS Admin Console no longer includes the Node API pod. The functionality has been migrated into the existing Go API pod, and the resulting release bundle size is ~33% smaller and contains 1 fewer pod. All upgrades will remove this pod.
+  * Helm render errors will now be included in the skippedFiles directory in the View Files tab.
+  * KOTS will no longer require a namespace when configuring the registry settings, allowing for anonymous registries.
+  * Preflight checks using the [textAnalyze](https://troubleshoot.sh/docs/analyze/regex/) function can now run against multiple files when the provided filepath is a [glob](https://golang.org/pkg/path/filepath/#Glob).
+  * Preflight checks relying on the [storageClass](https://troubleshoot.sh/docs/analyze/storage-class/) function can check for the presence of a default storageclass by leaving `storageClassName` empty.
+{{</changes>}}
+
+{{<fixes>}}
+  * Empty files will no longer be included in the skippedFiles directory in the View Files tab.
+  * Fixed support of `helm.sh/hook-delete-policy` annotation. Fixed transpiling to `kots.io/hook-delete-policy`.
+  * Fixed a bug that caused Admin Console pods to fail with `ImagePullBackOff` errors after an [upgrade in airgapped installs](/kotsadm/updating/updating-admin-console/#airgap-installations).
+  * Fixed an issue where some errors were not logged when restoring a snapshot.
+  * Fixed a bug that caused [`kots upstream upgrade`](/kots-cli/upstream/) command to fail with error 500.
+  * Fixed a bug that caused license installation to fail if app had missing or malformed [Application spec](/reference/v1beta1/application/).
+  * Fixed a bug that caused KOTS to re-deploy the first release of the application when re-running preflight checks for that release.
+  * Empty files are no longer included in yamlErrors.
+  * Fixed a bug that caused the user to be redirected to the wrong version after editing the registry settings and clicking the "View Files" tab.
+  * Fixed a bug that caused the `kots upload` command to fail with an empty KOTS config.yaml.
+  * Fixed a bug that caused the registry settings configuration to fail when entering ECR registry credentials.
+  * Fixed a bug that could cause `panic: d.nx != 0` error when images were being pushed to private registry during airgapped installs.
+  * Fixed an unclear error message when failing to parse an image ref while checking whether images were private. This error message will now include the filename that the offendin image was found within.
+  * Updated the generated `kustomization.yaml` when using a private registry to avoid a problem when two different tags of the same image are used.
+  * Fixed a bug where previously deployed versions in Kotsadm "Version Vistory" tab show a status of "Deployed" instead of "Previously Deployed".
```

Comment:
```suggestion
  * Fixed a bug where previously deployed versions in KOTS Admin Console "Version Vistory" tab show a status of "Deployed" instead of "Previously Deployed".
```